test/inductor/test_torchinductor.py

```diff
@@ -5756,6 +5756,10 @@ def fn(a):
 
         self.common(fn, (torch.randn([2, 4, 37, 38, 39]),))
 
+    @unittest.skipIf(
```

Comment:
IMO xfail might be more appropriate here. On a future pin update when the issue is resolved we will get signal that the test is passing.

Reply:
Good point. I could use `@unittest.expectedFailure` but didn't figure out how to set it conditionally. Do you have any pointers? Thanks.

Reply:
you have to write your own decorator, we hav lots of them but I don't think there is one for this yet. 


```python
def xfailIfSM90(fn):
    if torch.cuda.is_available() and torch.cuda.get_device_capability() == (9, 0):
        return unittest.expectedFailure(fn)
	return fn
```